### DISCUSSION TITLE
Where is `requiredStatusChecks` documented?

### DISCUSSION THREAD

**Discussion by adyanth** [renovatebot/renovate#28461] [Request Help] [answered]
### What would you like help with?

None

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I cannot seem to find any (latest) documentation on `requiredStatusChecks`. Even the [schema](https://docs.renovatebot.com/renovate-schema.json) does not list it. What gives?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
It has been deprecated. Use [`ignoreTests`](https://docs.renovatebot.com/configuration-options/#ignoretests) instead

**Reply by vjm** [reply]
do you have to be on a paid github tier to use `ignoreTests` then? I don't think branch protection rules for private repos are respected unless you're on a paid tier.... @RahulGautamSingh

**Reply by rarkins** [reply]
This feature is unrelated to GitHub tiers
